doc/simd_api.rst

```diff
@@ -0,0 +1,948 @@
+SIMD classes for Arbor
+======================
+
+The purpose of the SIMD classes is to abstract and consolidate the use of
+compiler intrinsics for the manipulation of architecture-specific vector
+(SIMD) values.
+
+The implementation is rather loosely based on the data-parallel vector types
+proposal P0214R6 for the C++ Parallelism TS 2.
```

Comment:
I think a mention of, and link to, the VC library would be good here.

Reply:
I'd rather add a link to the proposal: while certainly the genesis of the proposal is the Vc library, it has its own life (and the code here is based on the proposal, not Vc).